### PR TITLE
Add code for Hamburger menu icon

### PR DESCRIPTION
**Pull request by Amanulla**
Here I added code for Hambergar menu icon, and added menu items like 'portfolio, about, and contact' 
I gave Id's for three menu items, because when the user clicks on the menu item then it will scroll the webpage to a particular point.

### CODE REVIEW

File: hamburger.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    z-index: 10;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    padding-left: 10px;
+    background: #6070ff;
+    opacity: 0.97;
+    transition: top 1s ease 0s;
+    display: none;
+  }
```
Comment (vechicin) [inline hamburger.css]: Your mobile menu design doesn't quite match the template design.

Your design:

Screen Shot 2022-06-22 at 8 38 50

Template design:

Screen Shot 2022-06-22 at 8 39 16

Please fix the font so it matches that of the template design.

Screen Shot 2022-06-22 at 8 40 16

Additionally, according to the principles of mobile-first design, you should only use media queries to make a mobile version responsive to a desktop version. This media query is doing the opposite. Please remove it.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Amanulla ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Proper mobile menu implementation 
- [x] Nice transitions!

## Required Changes ♻️

- [ ] **NOTE FOR FELLOW REVIEWERS**: Stylelint checks are passing locally, please do not invalidate submission do to Stylelint errors

- [ ] Your desktop version is not working as intended. Please fix it. I am using Google Chrome, for reference.

Screen Shot 2022-06-22 at 8 44 07

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: hamburger.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    right: 1px;
+    left: 1px;
+    z-index: 10;
+    flex-direction: column;
+    padding-left: 35px;
+    background: #6070ff;
+    opacity: 0.9;
+    transition: top 1s ease 0s;
+    display: none;
+  }
```
Comment (leonardodiasb) [inline hamburger.css]: - [ ] Your `font-weight` still does not match with the Figma template. Kindly change that as the previous Code Reviewer pointed out.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Amanulla , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
...
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Amanulla   

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**